src/experimental/zookeeper/zookeeper/Chart.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: v1
+appVersion: "1.0"
```

Comment:
`appVersion` is for the wrapper app, i.e. zookeeper version. We can start with the latest zk 3.6.1.